modules/metering-cluster-usage-examples.adoc

```diff
@@ -24,4 +24,4 @@ spec:
 <1> To say organized, remember to change the name of your Report if you change any of the other values.
```

Comment:
s/say/stay

Reply:
This also needs to be updated in the "Measure cluster utilization using cron expressions" section too.